xbmc/threads/platform/pthreads/ThreadImpl.cpp

```diff
@@ -54,88 +54,103 @@ namespace XbmcThreads
     return true; // note, we never call destroy.
   }
 
-  static bool recursiveAttrSet = setRecursiveAttr();
+  static bool recursiveAttrSet = SetRecursiveAttr();
 
   pthread_mutexattr_t* CRecursiveMutex::getRecursiveAttr()
   {
     if (!recursiveAttrSet) // this is only possible in the single threaded startup code
-      recursiveAttrSet = setRecursiveAttr();
+      recursiveAttrSet = SetRecursiveAttr();
     return &recursiveAttr;
   }
   // ==========================================================
 }
-void CThread::SpawnThread(unsigned stacksize)
+
+static pid_t GetCurrentThreadPid_()
 {
-  pthread_attr_t attr;
-  pthread_attr_init(&attr);
-#if !defined(TARGET_ANDROID) // http://code.google.com/p/android/issues/detail?id=7808
-  if (stacksize > PTHREAD_STACK_MIN)
-    pthread_attr_setstacksize(&attr, stacksize);
+#ifdef TARGET_FREEBSD
+#if __FreeBSD_version < 900031
```

Comment:
no need for this, FreeBSD 9.0 is already EOL
seems like you reintroduced this during rebasing, I already removed it at https://github.com/xbmc/xbmc/pull/13595/commits/e658aa1816444dfc38cff5060f5726784a8bb66b from master

Reply:
Okay. Addressed here: https://github.com/xbmc/xbmc/pull/13721/commits/11b75373f4985a02a3b567e004e3b16bb041439c

Reply:
squash the removal into this commit for a clean history